src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -544,6 +567,17 @@ private string UFormatDateString(DateTime dateTime)
         }
 
         #endregion
+
+        private static class ParameterSetNames
```

Comment:
Our common patter is to use consts without extra static class.

Reply:
Did it.